debezium-server-http/src/main/java/io/debezium/server/http/HttpChangeConsumer.java

```diff
@@ -38,6 +39,7 @@
 import io.debezium.engine.DebeziumEngine;
 import io.debezium.server.BaseChangeConsumer;
 import io.debezium.server.http.jwt.JWTAuthenticatorBuilder;
+import io.debezium.server.http.standard_webhooks.StandardWebhooksAuthenticatorBuilder;
```

Comment:
We prefer to not have underscore in the package name. Maybe just `webhooks` is fine.